themes/classic/modules/ps_searchbar/ps_searchbar.tpl

```diff
@@ -24,9 +24,11 @@
  *}
 <!-- Block search module TOP -->
 <div id="search_widget" class="search-widget" data-search-controller-url="{$search_controller_url}">
-	<form method="get" action="{$search_controller_url}">
+	<meta itemprop="url" content="{$urls.pages.index}"/>
```

Comment:
Good job, is this something you can implement on the ps_searchbar module https://github.com/PrestaShop/ps_searchbar

Reply:
Hi @PierreRambaud 
This fix directly depends on the modification in `themes/classic/templates/layouts/layout-both-columns.tpl` (we need to implement this schema only on the homepage, according to Google documentation). Without it, it would yield validation errors. As the module tpl file is anyway overridden by the themes .tpl file, I think the safest way is to make this modification in the theme only.

Reply:
Now that I think about it, I can keep it in a single file... :)